.github/actions/deploy-klt-on-cluster/action.yml

```diff
@@ -7,6 +7,11 @@ inputs:
    k3d-version:
      required: true
      description: "Version of k3d that should be used"
+   k3s-version:
+     required: false
+     description: "Version of k3s that should be used"
+     # renovate: datasource=github-releases depName=k3d-io/k3d
```

Comment:
This should point to the k3s repo if you wanna have the k3s image updated. And it should probably use dockerhub as a datasource and not github-releases.
You can find the supported datasources for renovate [here](https://docs.renovatebot.com/modules/datasource/#supported-datasources)